browser/brave_browser_process_impl.cc

```diff
@@ -207,6 +208,18 @@ BraveBrowserProcessImpl::ad_block_regional_service_manager() {
   return ad_block_regional_service_manager_.get();
 }
 
+NTPSponsoredImagesComponentManager*
```

Comment:
each component does not need its own component manager. You  need a component installer policy